dnf+

```diff
@@ -0,0 +1,19 @@
+---
+tags: [ packaging ]
+---
+# To install a package:
+dnf install <package>
```

Comment:
Multiple fixes available: please chose one of them.

```suggestion
# Install a specific package:
dnf install "<package>"
```

- Optional change: add sample input and output right after this example. It's not a TlDr page where u have a more restricted and compact page format.
- Optional change: it's safer to use quotes.
- Optional change: `to` can be safely removed.
- Optional change: `a specific` may sound better.

```suggestion
# Install specific packages:
dnf install "<package1 package2 ...>"
```

- Almost the same as above.
- Optional change: if `dnf` can accept multiple packages at once for `install` command then u can use this fix.